### ISSUE TITLE
FlightGear flatpak fails to update

### ISSUE THREAD

**Issue by astromatt42** [flathub/org.flightgear.FlightGear#78]
Linux Mint 21.1, attempting to update FlightGear:

flatpak update
Looking for updates…

        ID                          Branch   Op  Remote   Download
 1. [✗] org.flightgear.FlightGear   stable   u   flathub  1.0 kB / 2.0 GB

Error: While pulling app/org.flightgear.FlightGear/x86_64/stable from remote flathub: URI https://dl.flathub.org/repo/deltas/mB/KTM2lMRAZvCR7YCzGcn6BeZHhUpaFnOQ43ZLo19NY-yp6N3N6cSc8hof6Ux+ZN_Vwjsd11feoyzIAUWkbYVp4/2 exceeded maximum size of 23389 bytes
Updates complete.
error: There were one or more errors

then:
flatpak update
Looking for updates…

        ID                          Branch   Op  Remote   Download
 1. [✗] org.flightgear.FlightGear   stable   u   flathub  1.0 kB / 2.0 GB

Error: While pulling app/org.flightgear.FlightGear/x86_64/stable from remote flathub: Server returned status 404: Not Found
Updates complete.
error: There were one or more errors

Other flatpaks update without issue.

**Comment by Vanadiae** [comment]
I just updated FlightGear on my side and it worked just fine. An update was merged recently, but it was two days before. Maybe Flathub had a hiccup when you tried updating. Looking at status.flathub.org, it looks like the [most recent event](https://status.flathub.org/issues/2023-01-14-deltas/) (in January) was similar to your issue, where you'd get either a 404 or a "exceeded maximum size of […]" error, so maybe try updating FlightGear with `--no-static-deltas` this time around as suggested there.
